### PR TITLE
It's an "upgrade", not a "potion"

### LINKED ISSUE #25370: Mining Drone AI upgrade is referred to as sentience potion
Server revision compiled on: 2017-03-21
37a325d8971f75136aae2c7f87c74b94ff12d9c9 

When using a mining drone AI upgrade on a minebot, it is referred to as a sentience potion.

Message: "You offer the sentience potion to the nanotrasen minebot..."

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
fixes #25370

### CODE REVIEW

**Comment by kevinz000** [comment]
You're an upgrade!~

File: code/modules/research/xenobiology/xenobiology.dm [hunk 1]
```diff
@@ -177,12 +177,12 @@
 		return ..()
 	var/mob/living/simple_animal/SM = M
 	if(SM.sentience_type != sentience_type)
-		to_chat(user, "<span class='warning'>The potion won't work on [SM].</span>")
+		to_chat(user, "<span class='warning'>\The [name] won't work on [SM].</span>")
```
Comment (Fox-McCloud) [inline code/modules/research/xenobiology/xenobiology.dm]: `[src] won't work on [SM]` would be much better.

Comment (ExcessiveUseOfCobblestone) [inline code/modules/research/xenobiology/xenobiology.dm]: what is the difference?

Comment (Supermichael777) [inline code/modules/research/xenobiology/xenobiology.dm]: its an animal thing

Comment (ExcessiveUseOfCobblestone) [inline code/modules/research/xenobiology/xenobiology.dm]: but [src]/[name] is the potion, not the animal.

Comment (XDTM) [inline code/modules/research/xenobiology/xenobiology.dm]: [src] auto-adds the The

Comment (ExcessiveUseOfCobblestone) [inline code/modules/research/xenobiology/xenobiology.dm]: Ahhhhh

**Pull request merged**